lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -99,7 +107,12 @@ function _signup_resource_access() {
  * @param boolean $competition
  * @return array
  */
-function _signup_resource_index($user, $campaigns, $count, $page, $competition) {
+function _signup_resource_index($user, $users, $campaigns, $count, $page, $competition) {
+
+  if (isset($users)) {
+    $user = $users;
```

Comment:
We can still deprecate the singular version if we wanted, but by doing 3 lines of code here I save myself from changing user --> users in many more places :wink: